api/src/main/scala/com/tersesystems/echopraxia/plusscala/api/ValueTypeClasses.scala

```diff
@@ -145,20 +145,24 @@ trait ValueTypeClasses {
       override def toAttributes(value: Value[_]): Attributes = Attributes.empty()
     }
 
-    implicit def iterableValueFormat[TV: ToValueAttributes]: ToValueAttributes[Iterable[TV]] = new ToValueAttributes[Iterable[TV]]() {
-      override def toValue(seq: collection.Iterable[TV]): Value[_] = {
-        val list: Seq[Value[_]] = seq.map(el => implicitly[ToValueAttributes[TV]].toValue(el)).toSeq
-        Value.array(list.asJava)
+    // Things that modify how values appear should apply to seqs and other packaging.
+    implicit def iterableValueFormat[TV: ValueModificationToValueAttributes]: ToValueAttributes[Iterable[TV]] =
```

Comment:
Things that involve modifying the value's presentation should still work even if in a Seq